lib/api/controllers/documentController.js

```diff
@@ -1,30 +1,58 @@
+/*
+ * Kuzzle, a backend software, self-hostable and ready to use
+ * to power modern apps
+ *
+ * Copyright 2017 Kuzzle
+ * mailto: support AT kuzzle.io
+ * website: http://kuzzle.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 'use strict';
 
-let
-  BadRequestError = require('kuzzle-common-objects').errors.BadRequestError,
-  SizeLimitError = require('kuzzle-common-objects').errors.SizeLimitError,
-  assertHasBody = require('./util/requestAssertions').assertHasBody,
-  assertHasId = require('./util/requestAssertions').assertHasId,
-  assertBodyHasAttribute = require('./util/requestAssertions').assertBodyHasAttribute,
-  assertHasIndexAndCollection = require('./util/requestAssertions').assertHasIndexAndCollection,
-  assertBodyAttributeType = require('./util/requestAssertions').assertBodyAttributeType,
-  assertIdStartsNotUnderscore = require('./util/requestAssertions').assertIdStartsNotUnderscore,
+const
+  {
+    BadRequestError,
+    SizeLimitError,
+    PartialError
+  } = require('kuzzle-common-objects').errors,
+  {
+    assertHasBody,
+    assertHasId,
+    assertBodyHasAttribute,
+    assertHasIndexAndCollection,
+    assertBodyAttributeType,
+    assertIdStartsNotUnderscore
+  } = require('./util/requestAssertions'),
   Request = require('kuzzle-common-objects').Request,
-  Promise = require('bluebird'),
-  PartialError = require('kuzzle-common-objects').errors.PartialError;
+  Bluebird = require('bluebird');
 
 /**
+ * @class DocumentController
  * @param {Kuzzle} kuzzle
- * @constructor
  */
-function DocumentController (kuzzle) {
-  const engine = kuzzle.services.list.storageEngine;
+class DocumentController {
+  constructor(kuzzle) {
+    this.engine = kuzzle.services.list.storageEngine;
+    this.kuzzle = kuzzle;
+  }
 
   /**
    * @param {Request} request
    * @returns {Promise<Object>}
    */
-  this.search = function documentSearch (request) {
+  search(request) {
     assertHasIndexAndCollection(request);
 
     if ([',', '*', '+'].some(searchValue => request.input.resource.index.indexOf(searchValue) !== -1)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 101 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)